v2.0/restore.md

```diff
@@ -63,6 +63,12 @@ Restore Type | Parameters
 **Full backup** | Include only the path to the full backup.
 **Full backup + <br/>incremental backups** | Include the path to the full backup as the first argument and the subsequent incremental backups from oldest to newest as the following arguments.
 
+### Point-in-time Restore
+
+If the full or incremental backup was taken [with revision history](backup.html#backups-with-revision-history), you can restore the data as it existed at the specified point-in-time within the revision history captured by that backup. 
+
+If you do not specify a point-in-time, the data will be restored to the backup timestamp (that is, the restore will work as if the data was backed up without revision history).
```

Comment:
nit: Instead of a paranthetical, let's just separate the two sentences with a semi-colon.

> If you do not specify a point-in-time, the data will be restored to the backup timestamp; that is, the restore will work as if the data was backed up without revision history.